CMakeLists.txt

```diff
@@ -176,21 +177,48 @@ endif()
 #############################################################
 
 if (ENABLE_INSTALL)
-  install(TARGETS openstreetmap_cgimap
-          PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-          DESTINATION "bin")
+    include(GNUInstallDirs)
 
-  install(FILES openstreetmap-cgimap.1 DESTINATION share/man/man1)
+    set(CMAKE_INSTALL_DEFAULT_DIRECTORY_PERMISSIONS
+        OWNER_READ OWNER_WRITE OWNER_EXECUTE
+        GROUP_READ GROUP_EXECUTE
+        WORLD_READ WORLD_EXECUTE)
```

Comment:
I coped this over from the existing code, but is this really required? The correct permissions should already be set automatically.
(I changed `CPACK_INSTALL_DEFAULT_DIRECTORY_PERMISSIONS` to `CMAKE_INSTALL_DEFAULT_DIRECTORY_PERMISSIONS` which is also used by cpack by default).